src/foam/box/SessionClientBox.js

```diff
@@ -48,7 +48,7 @@ foam.CLASS({
         } else {
 
           // fetch the soft session limit from group, and then start the timer
-          if ( this.group.id !== "" && this.group.softSessionLimit !== 0 ) {
+          if ( this.group && this.group.id !== '' && this.group.softSessionLimit !== 0 ) {
             this.sessionTimer.startTimer(this.group.softSessionLimit);
```

Comment:
Unrelated to this PR, but what is the softSessionLimit and why do we need it?

Reply:
softSessionLimit present a dialog x time before real session limit prompting the user to click to stay signed in. 
So in Ablii, the idle timeout is set to 10 minutes and the prompt appears 1 minute before.